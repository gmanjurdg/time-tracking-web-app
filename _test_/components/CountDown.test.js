import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import CountDown from '../../components/CountDown';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<CountDown />);
});

describe('CountDown-Components', () => {
  it('Rendering CountDown component', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  
});