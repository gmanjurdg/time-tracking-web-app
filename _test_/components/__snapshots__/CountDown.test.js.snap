// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountDown-Components Rendering CountDown component 1`] = `undefined`;
